VARIABLES.md

```diff
@@ -567,8 +567,7 @@ Default:  "{{sasl_protocol if sasl_protocol == 'kerberos' else 'none'}}"
 ### zookeeper_quorum_authentication_type
 
 Authentication to put on ZK Server to Server connections. Available options: [mtls, digest, digest_over_tls].
-
-Default:  "{{ 'mtls' if zookeeper_ssl_enabled and zookeeper_ssl_mutual_auth_enabled else zookeeper_sasl_protocol }}"
+Default:  "{% if zookeeper_ssl_enabled and zookeeper_ssl_mutual_auth_enabled %}mtls{% elif zookeeper_sasl_protocol == 'digest' %}digest{% else %}none{% endif %}"
```

Comment:
digest_over_tls is missing. I think we need to update that in docs as well

Reply:
As of now
zk_sasl_protocol is in [kerberos/digest]
zk_client_auth is in [mtls/digest/digest_over_tls]
zk_quorum_auth is in [mtls/digest/kerberos]

So should we instead modify zk_sasl_protocol variable such that
zk_sasl_protocol is in [mtls/digest/digest_over_tls/kerberos]
And when we define client and quorum auth based on the sasl_protocol then handle the case so that they dont take a value outside their domain

Reply:
no, zk_quorum_auth includes digest_over_tls, here is the [PR](https://github.com/confluentinc/cp-ansible/pull/1015). `Server to server auth only working for digest-md5` in zookeeper_sasl_protocol refers to this value